icu4j/main/tests/core/src/com/ibm/icu/dev/data/cldr/units/_readme.txt

```diff
@@ -0,0 +1,6 @@
+# Test data for units: conversions and preferences
```

Comment:
Won't this get nuked when someone runs the clean target?

Reply:
Yes. So if I read you right, you'd like "clean" to leave the repository in the desired commit state. Do we need a "clean" target at all presently?

Reply:
Deleted "clean" and "install" targets for now, they're not useful at this time.

Left `clean-cldr-testdata` target, to me it looks like it behaves similarly to `clean-icu-data`.

Reply:
SGTM